Correcci√≥n de la entrega:

E
======================================================================
ERROR: test_semana_02 (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_semana_02
Traceback (most recent call last):
  File "/usr/lib/python3.10/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/home/mlopez/git/homework-checker/01_getting_data/tests/test_semana_02.py", line 4, in <module>
    import timeout_decorator
ModuleNotFoundError: No module named 'timeout_decorator'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)

